### ISSUE TITLE
The obnoxious issue of doing 'portable' right

### ISSUE THREAD

**Issue by beppe9000** [iridium-browser/tracker#195]
I see that somebody brought the problem to light already: #144 and [here](https://github.com/iridium-browser/tracker/issues/43#issuecomment-257540048).

I am not satisfied with the current situation since the proposed portable edition is not portable (as in "self contained software that can be moved to a different pc without losing data or functionality") despite being labeled as such. I was almost falling for that, the notice below the download link saved me.

Rename it to "without installer (NOT PORTABLE)" or do the portable part the right way:
 - Patch Chromium instead of forcing people to make shortcuts: Windows Taskabar Pinning does not work correctly and will pin the executable itself so when one clicks on the taskbar to open it will not honor the user data path.

 - Use a trigger file: many apps use this approach, so that whenever a file named "portable.txt" is found in the EXE directory the app will start in portable mode, storing data in a local subfolder rather than a system-wide one. Something like `if(file_exists(".\portable.txt")) user_data_path=".\UserData";`

I understand this could be complicated so I don't expect this to be fixed anytime soon.

I have been a victim of something like this while trying the Vivaldi (chromium based) browser's "standalone" version that actually stores some data but not all: somebody on their forum said that chrome/ium cannot be portable because it uses the system crypto provider to encrypt user data so if moved on another pc the extensions and saved passwords would be unusable lie happened to me. 

I don't know how this is handled here by Iridium but if it is so then whoever fixes this issue should consider that too.

I may sound harsh, but I'm only pissed at Vivaldi :)

**Comment by metapea** [comment]
+1

**Comment by nima-amin** [comment]
Indeed, it would be at least good to have the "user data" folder within the application directory, as it is the case in for example Portable Brave, Centbrowser, X-Chromium, Comodo Dragon, Iron, Opera, Slimjet, Vivaldi. The thing with losing the extensions wouldn't be solved by this but at least you would have all the files you need (including the bookmarks) in the same location.
